src/main/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjection.java

```diff
@@ -204,20 +209,18 @@ private StringBuilder getResponseFromPingCommand(String ipAddress, Supplier<Bool
     public ResponseEntity<GenericVulnerabilityResponseBean<String>> getVulnerablePayloadLevel6(
             @RequestParam(IP_ADDRESS) String ipAddress)
             throws ServiceApplicationException, IOException {
-
+        Supplier<Boolean> condition =
+                getSupplier(
+                        StringUtils.isNotBlank(ipAddress)
+                                && (IP_ADDRESS_PATTERN.matcher(ipAddress).matches()
+                                        || ipAddress.contentEquals("localhost")));
```

Comment:
lets make localhost a constant in constants file (we already have one org.sasanlabs.vulnerability.utils.Constants.java) and make both the testcase and here a reference to it.